core-bundle/contao/templates/twig/content_element/swiper.html.twig

```diff
@@ -90,10 +96,20 @@ Custom template
             (function() {
                 const swiper = document.querySelectorAll('.swiper');
                 swiper.forEach (el => {
-                    let opts = '';
+                    let opts = {
+                        speed: el.getAttribute('data-speed'),
+                        initialSlide: el.getAttribute('data-offset'),
+                        loop: el.hasAttribute('data-loop'),
+                    };
+
+                    let delay = el.getAttribute('data-delay');
+                    if (delay > 0) {
+                        opts['autoplay'] = { delay: delay };
+                    }
```

Comment:
You do not have to keep that - your original version was ok. Anyone who had customized their template will have the original code (that's why we need to keep the original data attributes as well).